app/Example.java

```diff
@@ -0,0 +1,10 @@
+import io.ably.lib.realtime.AblyRealtime;
+
+public class Example {
+
+	private static final String API_KEY = "INSERT-YOUR-API-KEY-HERE";
+
+	public void publishAndSubcscribe() {
```

Comment:
Typo, and like https://github.com/ably/tutorials/pull/2, this should be set up rather than just publishAndSubscribe.